launchflow/gcp/artifact_registry_repository.py

```diff
@@ -68,7 +68,9 @@ def __init__(
         self,
         name: str,
         format: Union[str, RegistryFormat],
-        location: Optional[str] = None,
+        location: Optional[
+            str
+        ] = None,  # TODO: Rename this to region and deprecate location
```

Comment:
I believe it's location in the API so I would leave it as is, I think it can be zonal or regional

Reply:
removed the todo